pgt/src/PGT/Output/TestSuite/examples/failure-on-extra-line-after-definitions.expected.parsed

```diff
@@ -1 +1 @@
-Failed reading: expected database object definitions but found an empty line
```

Comment:
Comment: We can also have another database object definition after a previous definition. This error message seems to imply we can only have one definition and then tests.